script/validate-data/index.ts

```diff
@@ -105,6 +104,9 @@ async function checkWorkflow(workflowPath: string, propertiesPath: string): Prom
       }
       
     }
+    if (! workflowPath.endsWith("blank.yml") && (! properties.categories || ! properties.categories.some(category => allowed_categories.some(ac => ac.toLowerCase() == category.toLowerCase())))) {
```

Comment:
The space after `!` is not usually I have seen. is this a guidance in this repo?